modules/manage/pages/kubernetes/decommission-brokers.adoc

```diff
@@ -189,55 +199,294 @@ NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
 ```
 ====
 
-In this example each broker has 12 cores available. If you plan to scale down to five broker, then you would have 40 cores available, which means that your cluster is limited by core count to 40K partitions (well above the current 3018 partitions).
+In this example each broker has 8 cores available. If you plan to scale in to five broker, then you would have 40 cores available, which means that your cluster is limited by core count to 40K partitions, which exceeds the current 3018 partitions.
 
-NOTE: To best ensure the stability of the cluster, keep under 50K partitions per cluster.
+NOTE: To best ensure the stability of the cluster, maintain less than 50K partitions per cluster.
 
 === Decommission assessment
 
-The considerations tested above yield the following::
+The considerations tested above yield the following for the example case:
 
 * At least four brokers are required based on availability.
-* Cost is not a limiting factor in this example, but lower cost (and lower broker count) is always best.
-* At least 1.2TB of data resides on each broker (if spread across five brokers). This falls within the 1.5TB of local storage available in the example.
+* Cost is not a limiting factor in this example, but lower cost and lower broker count is always best.
+* At least 1.2TB of data resides on each broker when spread across five brokers. This falls within the 1.5TB of local storage available in the example.
 * At least five brokers are required based on the highest replication factor across all topics.
 * At 3018 partitions, the partition count is so low as to not be a determining factor in broker count (a single broker in this example environment could handle many more partitions).
 
-So the primary limitation consideration is the replication factor of five, meaning that you could scale down to five brokers at minimum.
+So the primary limitation consideration is the replication factor of five, meaning that you could scale in to five brokers at minimum.
```

Comment:
I notice that here you write it out (five), but in other contexts above ("8 cores"), you use a numeral. Wondering why--I'd make it consistent. I believe the style guidance is any number less than 9 is written out, and any over 9 you just write the numeral.